sw/host/cryptotest/testvectors/parsers/wycheproof_ecdsa_parser.py

```diff
@@ -71,6 +71,10 @@ def parse_test_vectors(raw_data):
                 # consumers may incorrectly handle large integers.
                 "qx": key["wx"],
                 "qy": key["wy"],
+                # Wycheproof does not provide tests for the `sign` operation,
+                # so we set the value of d (private key) to be zero as a
+                # placeholder.
+                "d": "00"
```

Comment:
The schema doesn't, but a previous version of the Rust test harness in this PR required `d` to be included for all test vectors. I recently changed this to do the argument validation later to remove this restriction, but I had only fixed this in the NIST test vector parser. This can now be removed.